cloudant-sync-datastore-core/src/test/java/com/cloudant/sync/datastore/DatastoreSchemaTests.java

```diff
@@ -405,4 +406,218 @@ public Integer call(SQLDatabase db) throws Exception {
             datastore.close();
         }
     }
+
+    /**
+     * Test that we can migrate an auto-generated complex database from schema version 100
+     * The database has the following characteristics:
+     * - 25 docs, for each doc:
+     * - Max tree depth 7
+     * - Multiple leaf nodes (conflicts)
+     * - Attachments
+     * - Deleted revs
+     * @throws Exception
+     */
+    @Test
+    public void migration100ComplexDatabaseWithoutDuplicatesTo200() throws Exception {
+        // Extract database to temp folder
+        File temp_folder = new File(TestUtils.createTempTestingDir(this.getClass().getName()));
+        File zippedWithDups = f("fixture/v100ComplexWithoutDuplicates.zip");
```

Comment:
I know this is just a copy/paste, but we should rename this field for this test to avoid confusion.